JSDemos/Demos/PivotGrid/VirtualScrolling/description.md

```diff
@@ -1 +1,7 @@
-The PivotGrid ships with virtual scrolling support. Virtual scrolling improves the performance when displaying large numbers of&nbsp;PivotGrid cells. The PivotGrid in&nbsp;this demo contains approximately 4000&nbsp;cells. To&nbsp;see virtual scrolling in&nbsp;action, scroll the PivotGrid horizontally.
\ No newline at end of file
+If the PivotGrid is bound to a large dataset, you can enable the virtual scroll mode to load only the cells that are in the viewport. This allows you to optimize data load time and reduces resources consumption. When users scroll the PivotGrid, it loads cells to be displayed in the viewport and removes other cells from memory.
+
+To allow users to scroll the PivotGrid virtually, set the scrolling.[mode](/Documentation/ApiReference/UI_Components/dxPivotGrid/Configuration/scrolling/#mode) to _"virtual"_.
+
+In this demo, the PivotGrid contains approximately 4000 cells. To see virtual scrolling in action, scroll the PivotGrid horizontally.
+
+To see virtual scrolling when the PivotGrid is bound to a remote storage, refer to the [Remote Virtual Scrolling](https://js.devexpress.com/Demos/WidgetsGallery/Demo/PivotGrid/RemoteVirtualScrolling) demo.
```

Comment:
Remote Virtual Scrolling is the next demo in the list. I believe there's no reason to give backlinks in this case.